### PR TITLE
Update Methods to Setup PSLab in Android Studio.

### LINKED ISSUE #2076: Requirement for New Ways To Set PSLab Android Project
**Actual Behaviour**
As of now the official README.md only contains one single method to set the project and build the PSLab project on Android Studio.

**Expected Behaviour**
There should be multiple ways to setup the project so that the conditions of all users are covered as what might work for one user might not always work for another one.

**Steps to reproduce it**
1) Open The FOSSASIA PSLab-Android Project on github.
2) Scroll down to the README.md file and open it.
3) On opening the file we can see that there is only one common way to setup the project using Android Studio.

**Screenshots of the issue**
Screenshot (449)

We can see that there is only one common way to setup the PSLab-Android Project.

**Would you like to work on the issue?**
Yes I would Like to work on this issue.

### PR DESCRIPTION
**Pull request by aryanguls**
Fixes #2076 

**Changes**: For the purpose of this request I have updated new PSLab Android Project Build methods to the README.md file to further aid new users to be able to setup the project and build it on Android Studio.

**Screenshot/s for the changes**:
Screenshot (446)
Screenshot (447)
Screenshot (448)

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

### CODE REVIEW

**Comment by aryanguls** [comment]
@mariobehling @hpdang Mentors please review.

File: README.md [hunk 1]
```diff
@@ -133,28 +133,47 @@ There are 2 flavors (build variants) of PSLab Android application.
...
+For setting up the PSLab android project you may follow any of the two methods listed below, that is, you may download the repository zip file or you may directly clone the repository to Android Studio.
+
+### By downloading the zip file
 
 1. Download the _pslab-android_ project source. You can do this either by forking and cloning the repository (recommended if you plan on pushing changes) or by downloading it as a ZIP file and extracting it.
 
 2. Open Android Studio, you will see a **Welcome to Android** window. Under Quick Start, select _Import Project (Eclipse ADT, Gradle, etc.)_
+@@ -159,6 +161,40 @@ To debug over Wi-Fi: http://blog.fossasia.org/android-app-debugging-over-wifi-fo
+Note :
+1. If you built your own hardware, change VendorID and/or ProductID in [CommunicationHandler.java](https://github.com/fossasia/pslab-android/blob/master/app/src/main/java/io/pslab/communication/CommunicationHandler.java)
```
Comment (mariobehling) [inline README.md]: use relative links

**Comment by aryanguls** [comment]
@mariobehling Sir I have added the relative links and have changed the numbering issue.

**Comment by aryanguls** [comment]
@mariobehling @hpdang @nikit19 Mentors please review and merge.

**Comment by CloudyPadmal** [comment]
Is this seem to be copied from #2003 ?

**Comment by aryanguls** [comment]
Yes sir I have taken the basic text from pull request #2003 and have added and made the required changes to the file as the earlier request had been open for 2 months but has been inactive since. Also I felt that this needed to be merged as soon as possible because I too had initially faced a problem because I was unaware of the other methods of setting up the project and felt the need for this to be merged. @CloudyPadmal

**Comment by aryanguls** [comment]
@CloudyPadmal Sir please review and merge as and when possible.

**Comment by aryanguls** [comment]
@CloudyPadmal @mariobehling @hpdang @nikit19 Mentors please review.

**Comment by aryanguls** [comment]
@mariobehling Sir please review this merge request as soon as possible.

**Comment by aryanguls** [comment]
@CloudyPadmal Sir since there hasn't been activity on this pull request by 2003 could you please go ahead and merge this request as soon as possible.

**Comment by NishithP2004** [comment]
Duplicate of #2003

**Comment by aryanguls** [comment]
@NishithP2004  I guess thats already been mentioned above along with the reason. The earlier request has been open since about 2 months with no activity and I feel this is an integral part of the README file and thus have added it. Mentors please merge.

**Comment by aryanguls** [comment]
Also the earlier merger was informed 10 days ago but no response has been received from him so far.

File: README.md [hunk 2]
```diff
@@ -133,31 +133,50 @@ There are 2 flavors (build variants) of PSLab Android application.
 Before you begin, you should already have the Android Studio SDK downloaded and set up correctly. You can find a guide on how to do this here: [Setting up Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio)
 
 ### Setting up the Android Project
+For setting up the PSLab android project you may follow any of the two methods listed below, that is, you may download the repository zip file or you may directly clone the repository to Android Studio.
```
Comment (mariobehling) [inline README.md]: android -> Android

**Comment by aryanguls** [comment]
@mariobehling Sir I have resolved the issue in the file.

File: README.md [hunk 3]
```diff
@@ -133,31 +133,50 @@ There are 2 flavors (build variants) of PSLab Android application.
...
+
+### By downloading the zip file
 
 1. Download the _pslab-android_ project source. You can do this either by forking and cloning the repository (recommended if you plan on pushing changes) or by downloading it as a ZIP file and extracting it.
 
-2. Open Android Studio, you will see a **Welcome to Android** window. Under Quick Start, select _Import Project (Eclipse ADT, Gradle, etc.)_
+2. Open Android Studio, you will see a **Welcome to Android** window. Under Quick Start, select _Import Project (Eclipse ADT, Gradle, etc.)To debug over Wi-Fi follow the steps given in this [Blog](http://blog.fossasia.org/android-app-debugging-over-wifi-fo)
+
+* **Note :**
+If you built your own hardware, change VendorID and/or ProductID in [CommunicationHandler.java](https://github.com/fossasia/pslab-android/blob/master/app/src/main/java/io/pslab/communication/CommunicationHandler.java)
```
Comment (mariobehling) [inline README.md]: Oh, make that link relative instead of absolute.

**Comment by aryanguls** [comment]
@mariobehling Sir I have resolved the changes by making the link relative.

**Pull request merged**